pygmt/src/ternary.py

```diff
@@ -66,6 +74,12 @@ def ternary(self, data, **kwargs):
     {transparency}
     """
     kwargs = self._preprocess(**kwargs)  # pylint: disable=protected-access
+
+    alabel = str(alabel) if alabel is not None else "-"
+    blabel = str(blabel) if blabel is not None else "-"
+    clabel = str(clabel) if clabel is not None else "-"
+    kwargs["L"] = "/".join([alabel, blabel, clabel])
```

Comment:
If all of alabel/blabel/clabel is None, would that mean an argument like `-L-/-/-` is still passed to GMT? Probably gives the same plot, but thinking whether a if-then statement that checks if any of alabel/blabel/clabel is not None would be good.

Reply:
An additional if statement or replace the currently used ones @weiji14 ?

Reply:
I assume this would be a new if statement, something like:
```
if alabel or blabel or clabel:
    alabel = str(alabel) if alabel is not None else "-"
    blabel = str(blabel) if blabel is not None else "-"
    clabel = str(clabel) if clabel is not None else "-"
    kwargs["L"] = "/".join([alabel, blabel, clabel])
```